src/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.png';
```

Comment:
what about the images having extension svg ?

Reply:
I've added the modules